brew_view/specification.py

```diff
@@ -141,6 +141,37 @@
         },
     },
 
+    "auth": {
+        'type': 'dict',
+        'items': {
+            "enabled": {
+                "type": "bool",
+                "default": False,
+                "description": "Use role-based authentication / authorization",
+            },
+            "token": {
+                'type': 'dict',
+                'items': {
+                    "algorithm": {
+                        "type": "str",
+                        "default": "HS256",
+                        "description": "Algorithm to use when signing tokens",
+                    },
+                    "lifetime": {
+                        "type": "int",
+                        "default": 1200,
+                        "description": "Time (seconds) before a token expires",
+                    },
+                    "secret": {
+                        "type": "str",
+                        "required": False,
```

Comment:
Maybe I'm missing something, but does this get auto-generated if it is not set?

Reply:
Nope. This is one of those cases where required needs to be true if auth is enabled.

We could have something like ChangeMe as the default I guess? I'd almost rather it fail though, because then they'll be forced to set it. If we had a default and they didn't change it that would be pretty bad.